packages/flutter/lib/src/physics/spring_simulation.dart

```diff
@@ -37,16 +44,26 @@ class SpringDescription {
     double ratio = 1.0,
   }) : damping = ratio * 2.0 * math.sqrt(mass * stiffness);
 
+  /// The greater the mass, the larger the amplitude of oscillation,
+  /// and the slower the speed of returning to the equilibrium position.
+  ///
   /// The mass of the spring (m). The units are arbitrary, but all springs
   /// within a system should use the same mass units.
   final double mass;
 
+  /// Stiffness defines the spring constant, which measures the strength of
+  /// the spring. A stiff spring applies more force to the object that is
+  /// attached when the spring is not at the rest position.
+  ///
   /// The spring constant (k). The units of stiffness are M/T², where M is the
   /// mass unit used for the value of the [mass] property, and T is the time
   /// unit used for driving the [SpringSimulation].
   final double stiffness;
 
-  /// The damping coefficient (c).
+  /// The damping coefficient (c), A pure number without physical meaning,
+  /// describes the oscillation and decay of a system after being disturbed.
+  /// The larger the damping, the fewer oscillations and smaller the amplitude
+  /// of the elastic motion.
```

Comment:
I would leave the original first sentence and keep it as its own paragraph, then make your additions a second paragraph (making sure your first sentence is a complete sentence). Something like:

> The damping coefficient (c).
>
> It is a pure number without physical meaning that describes the oscillation and decay of a system after being disturbed. The larger the damping...

Reply:
Thanks, I think this might make the structure clearer. I've reorganized the parameters for mass, stiffness, and damping accordingly.